### PR TITLE
Fix PR-Check cleanup job

### PR DESCRIPTION
**Pull request by ScrewTSW**
### What does this PR do?
Fixes PR-Check cleanup job that runs once a day.

### What issues does this PR fix or reference?
https://github.com/redhat-developer/che-functional-tests/issues/372

### How have you tested this PR?
local debugging.

### CODE REVIEW

**Comment by rhopp** [comment]
This PR has to wait for https://github.com/redhat-developer/rh-che/pull/962/files to be merged.

**Comment by ScrewTSW** [comment]
Almighty PR merged. Change is ready to be merged into rh-che as well.

File: .ci/cico_cleanup_dev_cluster.sh [hunk 1]
```diff
@@ -12,26 +12,38 @@
...
 else
   echo "OpenShift login failed"
-  echo "login: |${RH_CHE_AUTOMATION_RDU2C_USERNAME:0:1}*${RH_CHE_AUTOMATION_RDU2C_USERNAME:7:2}*${RH_CHE_AUTOMATION_RDU2C_USERNAME: -1}|"
-  echo "passwd: |${RH_CHE_AUTOMATION_RDU2C_PASSWORD:0:1}***${RH_CHE_AUTOMATION_RDU2C_PASSWORD: -1}|"
   exit 2
 fi
 
 PullRequests=$(curl -s https://api.github.com/repos/redhat-developer/rh-che/pulls?state=open | jq '.[].number')
+if [ $? -eq 0 ]; then
+  echo "Open PR get success."
```
Comment (Katka92) [inline .ci/cico_cleanup_dev_cluster.sh]: This message is confusing for me - sounds like you are getting 1 open PR. Something like "Getting list of opened PRs successful." should be better (same for fail message). WDYT?

Comment (ScrewTSW) [reply .ci/cico_cleanup_dev_cluster.sh]: can reword the message.

**Pull request merged**